composer.json

```diff
@@ -165,6 +165,7 @@
     },
     "config": {
         "allow-plugins": {
+            "php-http/discovery": false,
```

Comment:
This line is related to #50225, to make it easy to switch from one branch to another.